src/K1DModel/tendency.jl

```diff
@@ -39,18 +40,30 @@ end
     FT = eltype(q_tot)
     S_Nl::FT = FT(0)
 
+    if N_aer < eps(FT)
+        return S_Nl
+    end
+
     q = TD.PhasePartition(q_tot, q_liq, q_ice)
     S::FT = TD.supersaturation(thermo_params, q, ρ, T, TD.Liquid())
 
     (; r_dry, std_dry, κ) = kid_params
     w = ρw / ρ
 
+    if common_params.open_system_activation
+        _aerosol_budget = common_params.prescribed_Nd
+        _already_activated_particles = N_liq
+    else
+        _aerosol_budget = N_aer
+        _already_activated_particles = FT(0)
```

Comment:
I thought the difference between the open and closed system should be that in the closed system you subtract the number of activated cloud droplets from.N_aer. And in the open system you keep it fixed and the prescribed number of aerosol particles

Reply:
I think that the ways you thought of the open activation is different.  @sajjadazimi : when Anna and I checked the code to try to solve the problem, we did it by commenting out the lines in which we subtract the number of activated cloud droplets from N_aer, which results in not having aerosol tendencies. 

Of course, the two 'versions' of the open_activation produce different results for my cases. Since I don't know a lot about the aerosol activation parts of the model, I honestly don't know which way is the best one!
Both open_activation versions lead to N_liq values that are not halved anymore (in contrast with the closed case), they both work for my cases and don't cause any problem.